examples/trip_planner/use_planner.py

```diff
@@ -60,24 +59,26 @@ def choose_provider(model_type, model_name):
 
 def inference(input_data: dict, model_type="gpt", model_name="gpt-4o-mini") -> dict:
     llm_agent = choose_provider(model_type, model_name)
-    http_connection_serp = ScaleSerp()
-    tool_search = ScaleSerpTool(connection=http_connection_serp)
+    http_connection_search = Exa()
```

Comment:
Agent with Exa tool works worse. Oftenly returns irrelevant responses about weather and flight cost (With Exa have it runs twice longer that with Scale Serp). Maybe something needs to be tuned in Tool parameters. Or this Tool is just not the best choice for this case.